cost/google/idle_compute_instances/google_idle_compute_instances.pt

```diff
@@ -209,77 +216,102 @@ script "js_calculated_utilization", type: "javascript" do
   result "results"
   parameters "ds_compute_utilization", "ds_memory_utilization"
   code <<-EOS
-  results = []
-  for ( i =0; i < ds_compute_utilization.length; i++ ){
+  var results = []
+  for (i = 0; i < ds_compute_utilization.length; i++) {
```

Comment:
This should make use of _.each from the Underscore.js library: https://flexera.atlassian.net/wiki/spaces/FP/pages/2184937511/Underscore.js+101#_.each

Reply:
@XOmniverse thank you for this, I addressed this change